gestor_contactos_dpf22/gestor_contactos/crud.py

```diff
@@ -0,0 +1,54 @@
+from gestor_contactos.models import contacts
+from utils.datetime_utils import get_timestamp
+# from utils import datetime_utils
+
+def add_contacts(cedula,nombre,apellido,celular)->tuple[bool,str]:
+    for contacto in contacts:
+        if contacto['cedula'] == cedula:
+            return False, "Ya existe un contacto con esa cédula."
+        
+    contacts_nuevos={
+        'cedula':cedula,
+        'nombre':nombre,
+        'apellido':apellido,
+        'celular':celular,
+        'fecha_creacion':get_timestamp(),
+        }
+    
+    contacts.append(contacts_nuevos)
+    return True, f"Contacto agregado exitosamente: {contacts_nuevos}"
+    
+
+def get_contact(cedula)->tuple[bool,str]:
+    for contacto in contacts:
+        if contacto['cedula'] == cedula:
+            print("Información del contacto:")
+            print(f"Cédula: {contacto['cedula']}")
+            print(f"Nombre: {contacto['nombre']}")
+            print(f"Apellido: {contacto['apellido']}")
+            print(f"Celular: {contacto['celular']}")
+            return True, "Completado"
+    return False,f"No se encontró un contacto con esa cédula."
+
+
+def update_contact(cedula,nombre=None,celular=None,apellido=None)->tuple[bool,str]:
+    for contacto in contacts:
+        if contacto['cedula'] == cedula:
+            nuevo_valor=input("Ingrese el nuevo valor: ")
+            contacto['cedula'] = nuevo_valor
```

Comment:
Al ser cedula una clave o un registro con el cual identificaremos a una persona, es de vital importancia que este no sea alterado o cambiado una vez que nosotros hemos registrado a un usuario, puedes eliminarlo, pero evita a toda costa actualizar los datos que identiquen a otros datos, en este caso cédula ayuda a identificar a personas, entonces, no debería ser alterado, los otros registros sí:
```suggestion
            nuevo_nombre = input("Ingrese el nuevo nombre (opcional): ").strip()
            nuevo_apellido = input("Ingrese el nuevo apellido (opcional): ").strip()
            nuevo_celular = input("Ingrese el nuevo número celular (opcional): ").strip()

            if nuevo_nombre:
                contacto["nombre"] = nuevo_nombre
            if nuevo_apellido:
                contacto["apellido"] = nuevo_apellido
            if nuevo_celular:
                contacto["celular"] = nuevo_celular
```